documentation/docs/workflow.md

```diff
@@ -0,0 +1,142 @@
+The central idea is that every part of the version string conveys meaning. A major change introduces behavior that is incompatible with previous versions of the software, while a minor change adds backwards-compatible functionality and a patch simply fixes a problem. So just by looking at the version string an implementer will have an understanding of the effort needed to integrate a new version.
+For more detailed information and an overview of advanced features see [Semantic Versioning 2.0.0](https://semver.org/). Not to be confused with [Sentimental Versioning](http://sentimentalversioning.org/).
+
+### Changelog
+A changelog is a file describing a software project for humans to grasp the type and content of changes from version to version. Changelogs are closely related to the versioning of software, since individual changes are grouped into versions that are in our case referenced by a [SemVer string](#Semantic-Versioning). We generally follow the recommendations from [keepachangelog](https://keepachangelog.com/en/1.0.0/). The changelog in this project is generated from the titles and [labels](#PR-Labels) of [Pull-Requests](#Pull-Requests). 
+
+#### PR Labels
+Labels are used to categorize changes in [Pull-Requests](#Pull-Requests). Adding a label will include the labeled [Pull-Request](#Pull-Requests) in the related section of the generated [Changelog](#Changelog).
+
+Changelogs are generated for the core Rust library and each binding separately. To attach a PR to a specific changelog use the following labels:
+##### `Rust` 
+Includes the PR in the core Rust library changelog.
+
+##### `Wasm`
+Includes the PR in the Wasm bindings changelog.
+
+It is also necessary to add an appropriate label for the type of change in the PR. The following labels determine in which section a PR title will appear: 
+
+##### Changed
+Maps to the major version of [Semantic Versioning](#Semantic-Versioning).
+labels: `Breaking change`
+
+##### Added
+Maps to the minor version of [Semantic Versioning](#Semantic-Versioning).
+labels: `Added`
+
+##### Patch
+Maps to the patch version of [Semantic Versioning](#Semantic-Versioning).
+labels: `Patch`
+
+##### Deprecated
+Marks features that will be removed in the feature. No special version consideration apply here, since the feature did not change yet.
+labels: `Deprecated`
+
+##### Removed
+Marks features as being removed. Typically the features should have been deprecated in the previous version. This maps to the major version of [Semantic Versioning](#Semantic-Versioning). 
+labels: `Removed`
+
+##### Excluded Tags
+Marks changes that should not be part of the changelog. This should only be used for documentation and rare exceptions
+labels: `Documentation`, `No changelog`
+
+Please note that a PR can only be listed in one section of a changelog. So attaching the labels `Rust` `Added` `Patch` to a PR is invalid.
```

Comment:
```suggestion
Please note that a PR can only be listed in one section of a changelog. So attaching the labels `Rust` `Added` `Patch` to a PR, for example, is invalid because `Added` and `Patch` conflict.
```